src/main/java/frc/robot/Robot.java

```diff
@@ -36,8 +36,8 @@
     private Command autonomousCommand;
```

Comment:
Andrew, Ben told me to find typos, but I have no clue how to read this stuff. It's like a 3 year old proofing Shakespeare. It's probably great. Keep it up!